docs/weaver_code_anatomy.md

```diff
@@ -112,63 +123,70 @@ sections:
+1. This final variable is customized for your interview. It lets you trigger all the code in this entire code block. If you are including this interview in another interview, you can use this variable to trigger this particular question order. It is also triggered in the [main order block](#main-order)
+
+## Main Order
+This block controls the higher-level order of the interview, anything that isn't questions specific to your interview, like intro screens, signatures, etc. 
+You shouldn't have to change that code unless you are 🚧 combining multiple interviews together 🚧.
+
+```yml
+mandatory: True
+id: main_order_a_258e_motion_for_impoundment
+code: |
+  al_intro_screen
+  a_258e_motion_for_impoundment_intro
+  # Interview order block has form-specific logic controlling order/branching
+  interview_order_a_258e_motion_for_impoundment_intro
   signature_date
   # Save (anonymized) interview statistics.
   store_variables_snapshot(data={'zip': users[0].address.zip})
-  a_258e_motion_for_impoundment_preview_question # Pre-canned preview screen
+  a_258e_motion_for_impoundment_preview_question  # Pre-canned preview screen
   basic_questions_signature_flow
   users[0].signature
-  a_258e_motion_for_impoundment = True
+  a_258e_motion_for_impoundment_download
 ---
 ```
 
-There is some AssemblyLine code that comes after your own custom interview order code. You will probably leave this code alone as well:
-
-1. `set_parts(subtitle=str(users))` adds to the information a logged in user will listed for this interview in their list of interviews. For an attorney, they should see the name of their clients. For a self represented litigant, they should see their name.
-1. `set_progress()` changes the progress bar shown to the person who's interacting with the form. When they are at the beginning of the form, it should be empty. When they are at the end, other code will make sure it is full. The [ALWeaver](https://apps-test.suffolklitlab.org/start/assemblylinewizard/assembly_line/#/1&new_session=1) tries to handle intermediate values between those two places that will make sense to the user. The example interview is short, so intermediate progress is only set once.
-1. `signature_date` is needed on every form that we know of.
-1. You should be very thoughtful when you use `store_variables_snapshot()`. It lets you gather data about how your form is being used. Care must be taken to anonymize it. Just removing a name is not sufficient.
+1. `signature_date` is the date that the user signed the form, and is needed on every form that has a signature.
+1. `store_variables_snapshot()` lets you gather data about this interview session. _You should be very thoughtful  about what you store, and care must be taken to anonymize it_. Just removing a name is not sufficient.
```

Comment:
```suggestion
1. `store_variables_snapshot()` lets you gather data about this interview session. _You should be very thoughtful about what you store, and care must be taken to anonymize it_. Just removing a name is not sufficient.

If you want to avoid asking the user for their address, you will need to change the information you save here. This code forces the user's address to be asked.
```